ui/dashboards/src/components/AddPanelButton/AddPanelButton.tsx

```diff
@@ -11,19 +11,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Button } from '@mui/material';
+import { Button, ButtonProps } from '@mui/material';
 import AddPanelIcon from 'mdi-material-ui/ChartBoxPlusOutline';
 import { InfoTooltip } from '@perses-dev/components';
 import { TOOLTIP_TEXT } from '../../constants';
 import { useDashboardActions } from '../../context';
 
-export const AddPanelButton = () => {
+export interface AddPanelButtonProps extends Pick<ButtonProps, 'fullWidth'> {
+  /**
+   * The variant to use to display the button. The `primary` variant is used
+   * in toolbars. The `secondary` variant is used in the empty state messaging.
+   */
+  variant?: 'primary' | 'secondary';
```

Comment:
what do you think about calling this `outlined` instead of `secondary`?

Reply:
I was thinking `secondary` because it impacts both the color and the outlined/not status, so this variant is more the "use case" than "what it looks like." Maybe it'd be better to call it something slightly different to differentiate it from the MUI variant. Hmmmm.